andela/northstar-backend
# #168477780 Add "facility_id" To "bookings" Table

#### What does this PR do?
This bugs adds the `facility_id column` to the booking table
which was hitherto missing. And then, it reverts the functionality
of giving `rating/feedback to the checked_in column` on the bookings
table. It was formerly a stand-alone functionality.

#### How should this be manually tested?
An `npm test` should pass with zero errors. If you experience issues with migration; recreate your database Schema inside your `postgres GUI e.g. pgAdmin4` with the shortcut below. You can paste all commands in one go.
```
DROP SCHEMA public CASCADE;
CREATE SCHEMA public;
GRANT ALL ON SCHEMA public TO postgres;
GRANT ALL ON SCHEMA public TO public;
```

#### What are the relevant pivotal tracker stories?
[#168477780](https://www.pivotaltracker.com/story/show/168477780)
